java/code/src/com/suse/manager/hub/HubManager.java

```diff
@@ -1097,6 +1171,16 @@ public boolean synchronizeRepositories(IssAccessToken accessToken) {
         return ProductsController.doSynchronizeRepositories();
     }
 
+    /**
+     * Trigger a synchronization of Repositories on the peripheral
+     *
+     * @param user The current user
+     * @return a boolean flag of the success/failed result
+     */
+    public boolean synchronizeRepositories(User user) {
+        ensureSatAdmin(user);
+        return ProductsController.doSynchronizeRepositories();
+    }
```

Comment:
This is not needed. We have this already in a different namespace as `mgr-sync refresh` is using this API already.
We had to implement it here only for use via token API.